.github/workflows/ci_tests.yaml

```diff
@@ -146,7 +146,7 @@ jobs:
     # TODO: gatekeeper-operator does not --log-mutations nor --mutation-annotations
     - name: Remove unhandled gatekeeper bats tests
       run: |
-        sed -i -e '67,71d;80,83d' /home/runner/work/gatekeeper-operator/gatekeeper-operator/gatekeeper/test/bats/test.bats
+        sed -i -e '67,73d;82,85d' /home/runner/work/gatekeeper-operator/gatekeeper-operator/gatekeeper/test/bats/test.bats
```

Comment:
This is frightening. There are a couple of `TODOs` in here we should seek to clean up. I've created an issue for it: https://issues.redhat.com/browse/ACM-8840

Reply:
OH! I created https://issues.redhat.com/browse/ACM-8749 a long time ago. Did you intend this?

Reply:
good good!!